### DISCUSSION TITLE
kafka and zookeeper arm64_big_sur bottle build can be enabled

### DISCUSSION THREAD

**Discussion by mologie** [Homebrew/discussions#2322] [Everyday usage] [answered]
Hi, I found that `kafka` and its dependency `zookeeper` build and run on Apple Silicon, but no ARM bottle is available right now. Both formulas pass `brew test` and `brew audit --strict`, and I have tested the Kafka tooling against a local Kafka cluster of mine without issues.

Following the example of #1519 I'd kindly request bottle builds for both formulas to be enabled. Thank you!

**Comment by cho-m** [comment, marked as answer]
Thanks for letting us know. Bottles should be available now:
- `zookeeper` ARM bottle: https://github.com/Homebrew/homebrew-core/commit/4001ef48495cf56cfcb3b928236d308e85ba2fd0
- `kafka` ARM bottle: https://github.com/Homebrew/homebrew-core/commit/3b9538449e59122ab3675f46a80e393540c391b9
